frontend/src/components/Typography/Body/index.module.css

```diff
@@ -1,5 +1,5 @@
 .body {
-  font-family: var(--font-family-body);
+  font-family: var(--font-family-body), sans-serif;
```

Comment:
Hmm so ... would this be something to add to the global var?

Reply:
Added on the CSS constants pr.